bootstrap

```diff
@@ -28,6 +28,8 @@ export SAGE_ROOT
 cd "$SAGE_ROOT"
 
 export PATH="$SAGE_ROOT/build/bin:$PATH"
+# Faster than calling sage-package
+export SAGE_PACKAGE="env PYTHONPATH=$SAGE_ROOT/build $(sage-bootstrap-python -c 'import sys; print(sys.executable)') -m sage_bootstrap.cmdline"
 
```

Comment:
Is the speedup really significant enough to sacrifice readability of the code as in 
```diff
-    for pkgname in $(sage-package list :standard:); do
+    for pkgname in $(${SAGE_PACKAGE-sage-package} list :standard:); do
```
Why do you need the speedup?

Reply:
I'll check whether it's still needed after the last change.

Reply:
Not needed any more, I've taken it out.

Reply:
Not yet?

Reply:
There was one holdout. Fixed now.